### DISCUSSION TITLE
Custom Branding css and Js

### DISCUSSION THREAD

**Discussion by Muta-Jonathan** [AzuraCast/AzuraCast#6390] [General]
Hello folks .........
Have tried to use custom branding for updating the Footer for all pages and background for login and also title brand but am failing
tried something like this 
```
let footerElement = document.getElementById('footer');
  if (footerElement) {
    footerElement.innerHTML = '© 2023 Your Custom Brand';
  }
```
though seems not pulling for footer 
cc @BusterNeece

**Comment by BusterNeece** [comment]
Please don't mention me directly with these kinds of questions. We don't provide basic JavaScript/CSS support for our application (especially if your intention is to remove the attribution saying the piece of software is AzuraCast and incorrectly claim your own copyright to our code); you can find community support on [our Discord](https://discord.gg/azuracast).

**Reply by Muta-Jonathan** [reply]
oh thanks however my intetion is not to claim the copyRights
